test/MainEvent.cxx

```diff
@@ -175,6 +187,7 @@ int main(int argc, char **argv)
       nevent = TMath::Min(nevent,nentries);
       if (read == 1) {  //read sequential
          //by setting the read cache to -1 we set it to the AutoFlush value when writing
+         ioperf = new TTreePerfStats("Perf Stats", tree);
```

Comment:
The data collection takes 'some' time and thus if one wants to externally merge 'pure' performance one would want to run without.